build.gradle

```diff
@@ -44,6 +44,7 @@ configurations {
 
```

Comment:
We can remove the commented code here: https://github.com/opensearch-project/opensearch-sdk/pull/76/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R55-R63

Reply:
The link redirects to the same diff. Could you point me to the link on `main`?

Reply:
Not sure why the link redirects to the same diff. May be since it's not a part of this PR but here it is https://github.com/opensearch-project/opensearch-sdk/blob/main/build.gradle#L55-L62

Reply:
Sure done.